packages/fx-core/src/plugins/resource/localdebug/tasks.ts

```diff
@@ -9,6 +9,8 @@ export function generateTasks(
   includeFrontend: boolean,
   includeBackend: boolean,
   includeBot: boolean,
+  includeAuth: boolean,
```

Comment:
Is `includeAuth = !(isMigrateFromV1)`?

Reply:
Yes, now includeAuth = !(isMigrateFromV1).
Would you like to split includeAuth and isMigrateFromV1?
Maybe there will be no simple auth tab app in the future.

Reply:
Both ok to me, just to confirm.